### DISCUSSION TITLE
Release 2.0.0

### DISCUSSION THREAD

**Discussion by sunwire** [scsitape/stenc#94] [General]
Maybe it's time for the new release?

**Comment by jmwilson** [comment]
I think it's feature-complete for release. There are some "cosmetic" things I would like to make consistent (e.g. the use of "volume" or "current block" in the output and documentation), but haven't gotten around to trying some changes.

**Comment by jonasstein** [comment]
I suggest to close https://github.com/scsitape/stenc/issues/95 and https://github.com/scsitape/stenc/issues/69
before tagging 2.0 
It is not a big thing and licenses can not be clarified too early
The distributions need to know about the changed dependencies.

**Comment by jonasstein** [comment]
[2.0.0_rc is ready for testing](https://github.com/scsitape/stenc/discussions/110)
